src/main/scala/com/microsoft/hyperspace/actions/CreateActionBase.scala

```diff
@@ -49,8 +51,18 @@ private[actions] abstract class CreateActionBase(dataManager: IndexDataManager)
 
     val signatureProvider = LogicalPlanSignatureProvider.create()
 
+    // Resolve the passed column names with existing column names from the dataframe if possible.
+    val (resolvedIndexedColumns, resolvedIncludedColumns) = {
+      try {
+        // Try creating log entry with resolved column names.
```

Comment:
I think we should always use the casing that the user provided.

Reply:
changed this behavior to not handle unresolved case. if `resolve` fails, it will directly throw exception. No "CREATING" file gets created at this point.